### DISCUSSION TITLE
Reading reweighting weights with pylhe

### DISCUSSION THREAD

**Discussion by CoCarrivale** [scikit-hep/pylhe#200] [Q&A]
Dear experts,

I’d like to know if it’s possible to read the reweighting weights in LHE files using the current version of the package.
Thank you kindly,

Best regards
Costanza Carrivale

**Comment by eduardo-rodrigues** [comment]
Hello @CoCarrivale. Apologies for us being slow but we have all been at a conference. And thank you for the interest in the package!

You can see some weight related information being parsed, see (grep'ing for "weight" in) https://github.com/scikit-hep/pylhe/blob/main/src/pylhe/__init__.py. But I suspect that's not exactly what you are looking for, right? Then the short answer is yes it is possible, but there is some code to be added to the package, I'm afraid.

We very much welcome contributions. Is this something you would be willing to contribute? It does not seem like an awful lot of work, basing on what is already in the package.

Let us know :-).
